docs/vs-tools-tizen/tools/profiler-user-manual.md

```diff
@@ -1,137 +1,154 @@
-     ![Jump To Line](../image/jump_to_line.png)     
+﻿# Profiler
+
+The profiler allows you to analyze the performance of your application. You can record, manage, and view profiling sessions.
+
+## Recording Profiling Sessions
+
+To record a profiling session:
+1. In the Visual Studio menu, select **Tools &gt; Tizen &gt; Profiler &gt; Run Profiler**.  
+  The profiler configuration dialog opens.
+
+2. Define the profiling options.  
+    ![Profiler configuration](media/profiler-config.png)
+    - **Path to application** is automatically filled with the information for the current solution.
+    - **Profiling Preset**  
+    You can select from the following predefined profiling configurations:
+      - **CPU Sampling**  
+        Trace CPU and source lines, but not memory allocation.
+      - **Memory Allocation**  
+        Trace memory allocation and source lines, but not CPU.
+      - **Complex Profiling**  
+        Trace CPU, source lines, and memory allocation.
+    - **Sampling Options**  
+    You can set the following sampling options:
+      - **Sampling Interval**  
+        You can set the sampling time interval, in milliseconds. The default is 10 ms.
+      - **High Granularity**  
+        You can switch on high-granularity sampling. It enables better precision and sample regularity, at the cost of efficiency.
+    - **Trace Options**  
+    You can set the following trace options:
+      - **Trace Execution**  
+        You can trace CPU utilization.
+      - **Trace Memory**  
+        You can trace memory allocation for methods.
+      - **Trace Source Lines**  
+        You can switch on line source mapping.
+      - **Stack Track**  
+        You can enable a hybrid trace, which registers stack changes on memory allocation.
+    - **CPU Tracing Options**  
+    You can set the following CPU tracing options:
+      - **Enable CPU Tracing**  
+        You can switch on the CPU tracing features.
+      - **Trace Process**  
+        You can trace the application's CPU utilization.
+      - **Trace Threads**  
+        You can trace each processing thread's CPU utilization.
+      - **Trace Interval (ms)**  
+        You can set the sampling time interval, in milliseconds.
+    - **Misc Options**  
+    You can set the following additional option:
+      - **Delayed start**  
+        You can delay the application startup.
+
+3. To start the profiling session, click **OK**.  
+    The application is installed and launches on the emulator.  
+    The **Output** panel displays the installation log, including any errors.  
+    ![Output panel](media/profiler-output-window.png)
+
+    The **Profiling Progress** panel shows live CPU and memory usage information.  
+    ![Profiling progress](media/profiler-progress.png)
+
+4. To pause and resume profiling, in the **Profiling Progress** panel, click **Pause** and **Resume**.  
+    For the time interval when profiling is paused, the profiling chart shows a light-blue background.  
```

Comment:
What exactly does this mean? The light blue background is also visible after resuming.

Reply:
Text clarified